bin/browserstack-jasmine.js

```diff
@@ -1,5 +1,6 @@
 /* eslint no-console: 0 */
 
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Do you know why import/no-extraneous-dependencies has to be disabled here? browserstack-runner is defined as a dependency in package.json